OWNERS_ALIASES

```diff
@@ -2,6 +2,8 @@ aliases:
   sig-data-science-leads:
     - erikerlandson
     - michaelclifford
+  sig-services-leads:
+    - tumido
```

Comment:
any other volunteers?

Reply:
Let's discuss than during the first meeting along with the subprojects. For now, we should merge this as is, wdyt?